src/components/baseComponents/comments/CommentItem.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Grid, Box, SvgIcon, IconButton, Collapse } from '@material-ui/core';
+import { green, red } from '@material-ui/core/colors';
+import { withStyles, createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
+
+import sprite from "assets/img/icons-sprite.svg";
+
+import { CommentChildProps } from './CommentChild';
+
+import moment from 'moment';
+import ReplyCommentPanel from './ReplyCommentPanel';
+import { Comment } from 'model/comments';
+
+interface CommentItemProps {
+  comment: Comment;
+}
+
+const Stretch = withStyles({
+	root: {
+		flexGrow: 1
+	}
+})(Grid);
+
+const GreenRedTheme = createMuiTheme({
+	palette: {
+		primary: green,
+		secondary: red
```

Comment:
@william-gooch not sure about this. colors should be used in css file. if it is not popup or iframe